src/main/scala/org/constellation/util/HealthChecker.scala

```diff
@@ -112,44 +74,19 @@ class HealthChecker[F[_]: Concurrent](
 
   val snapshotHeightInterval: Int = ConfigUtil.constellation.getInt("snapshot.snapshotHeightInterval")
 
-  val snapshotHeightRedownloadDelayInterval: Int =
-    ConfigUtil.constellation.getInt("snapshot.snapshotHeightRedownloadDelayInterval")
-
-  def checkClusterConsistency(ownSnapshots: List[RecentSnapshot]): F[Option[List[RecentSnapshot]]] = {
+  def checkClusterConsistency(ownSnapshots: List[RecentSnapshot]): F[Unit] = {
```

Comment:
It doesn't check for cluster consistency anymore. It's just for clearing stale tips and nothing else.

Reply:
Yep feel free to change it